esphome/components/tca9548a/__init__.py

```diff
@@ -0,0 +1,25 @@
+import esphome.codegen as cg
+import esphome.config_validation as cv
+from esphome.components import i2c
+from esphome.const import CONF_ID, CONF_SCAN
+
+CODEOWNERS = ['@andreashergert1984']
+
+DEPENDENCIES = ['i2c']
+
+tca9548a_ns = cg.esphome_ns.namespace('tca9548a')
+TCA9548AComponent = tca9548a_ns.class_('TCA9548AComponent', cg.PollingComponent, i2c.I2CDevice)
```

Comment:
Shouldn't this be `I2CMultiplexer`?

```suggestion
TCA9548AComponent = tca9548a_ns.class_('TCA9548AComponent', cg.PollingComponent, i2c.I2CMultiplexer)
```